judges/quality-of-service/quality-of-service.rb

```diff
@@ -105,3 +105,19 @@ def lifetime(type)
   end
 end
 f.average_backlog_size = issues.empty? ? 0 : issues.inject(&:+) / issues.size
+
+# Rejection PR rate
+all_prs_count = 0
+unmerged_prs_count = 0
+Fbe.unmask_repos.each do |repo|
+  all_prs_count += Fbe.octo.search_issues("repo:#{repo} type:pr closed:>#{$SINCE.utc.iso8601[0..10]}")[:total_count]
+  unmerged_prs_count += Fbe.octo.search_issues(
+    "repo:#{repo} type:pr is:unmerged closed:>#{$SINCE.utc.iso8601[0..10]}"
+  )[:total_count]
+end
+f.average_pull_rejection_rate =
```

Comment:
@Yegorov if you make variable names shorter, you can write this expression in just one line

Reply:
@yegor256 no, 3 symbols left
```
judges/quality-of-service/quality-of-service.rb:114:121: C: Layout/LineLength: Line is too long. [123/120]
  rejected += Fbe.octo.search_issues("repo:#{repo} type:pr is:unmerged closed:>#{$SINCE.utc.iso8601[0..10]}")[:total_count]
                                                                                                                        ^^^
```

Reply:
@Yegorov you don't need to check for `rejected.zero?`, just a check for `pulls.zero?` is enough